src/components/AccessControl.vue

```diff
@@ -0,0 +1,265 @@
+<script>
+import { stringify } from 'qs';
+
+import { createLocationValidator } from '../utils';
+import Breadcrumb from './Breadcrumb.vue';
+import Search from './Search.vue';
+
+export default {
+  name: 'AccessControl',
+  components: {
+    Breadcrumb,
+    Search,
+  },
+  inject: ['girderRest'],
+  props: {
+    model: {
+      type: Object,
+      required: true,
+      validator: model =>
+        createLocationValidator(false)(model) && model._modelType !== 'user',
+    },
+    noActionButtons: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
OK, that makes sense.

I think it would still be good to enable/disable the save button and cancel button separately.  Perhaps the prop should be called `noSaveButton`?

Open to suggestions on how exactly to manage those.

Reply:
If we control them separately. Will it be a little odd to present the component in a way to just have the cancel button but no save button?

Reply:
The use cases I think should be supported:

* Save button + Cancel button
* Save button alone.
* Neither.

For lack of a better idea, I thought we could have:

* A prop to control the save button visibility
* A slot for the consumer to add a cancel button if they wanted, so that this component doesn't have to know anything about its context (dialog, wizard, whatever).

If a cancel button and no save button happened, that would be the consumer's fault.

Reply:
Thank you for the detail explanation. But I think the access control would mostly be used in a dialog, which most likely needs both buttons. 
When the context doesn't want to use the provided buttons, the context is free to create buttons outside of this component instead of injecting it into the component with a slot because specifically in this case, the placement of the buttons is at the bottom anyway.

Reply:
> Save button alone. 
Would it be a little harsh on the user? (Not providing a way out without modifying)

Reply:
I just wrote a response then accidentally closed the window while trying to comment.  Give me a few min to type it out again...

Reply:
My goal is to be as flexible as possible without adding maintenance cost.  If we need to support different things in the v-card-actions area in the future, that will be a breaking change.  Little breakages like that add up and become annoying.

Rather than use a prop to control visibility and risk API breakages as well as issues where you need to apply new styling to buttons, how about this:

```pug
slot(name="card-actions" v-bind="{ save, loading }")
  v-card-actions(v-if="!noActionButtons")
    v-spacer
    v-btn(text, @click="$emit('close')") Cancel
    v-btn(color="primary", depressed, @click="save") Save
```

This will remove the prop `noActionsButtons`, prevent future breakages, and support a much wider variety of future needs.

Reply:
> Not providing a way out without modifying

In this case, the accessControl is not inside a dialog, it's embedded directly on a page somewhere, so "cancel" would not have the same meaning.   

I agree that if this is in a dialog, it should always have a cancel button.

Reply:
@subdavis The bigger slot sounds great. It also provides the save in a different way. 🙂